### ISSUE TITLE
An option when I initialize the DB for `nullzero` by default?

### ISSUE THREAD

**Issue by frederikhors** [uptrace/bun#287]
I'm repeating over and over ``bun:",nullzero"``.

What do you think about an option for make it default?

Maybe like:

`dbBunPG := bun.NewDB(dbPG, pgdialect.New(), WithNullZero())`

**Comment by vmihailenco** [comment]
PR is welcome :)

**Comment by rustiever** [comment]
HI @vmihailenco , I'm interested in this  issue
when i browsed through the code base, i came to know i have to add functional opts in db.go file.

from my understanding i must write zero values to all struct fields of DB struct. Am i right?
if i'm wrong kindly guide me to approach this

**Comment by vmihailenco** [comment]
Hey @rustiever ,

This issue is about providing a default value for [nullzero](https://github.com/uptrace/bun/blob/master/schema/table.go#L355) option. While it is definitely possible, it is not trivial since the code lives in a sub-package and we don't allow to override defaults for other options so this would be a new precedent that requires a new mechanism to achieve this.

After some considerations I think we should NOT support this and I've just removed "good first issue" label. Sorry for misleading you.

My thinking is that allowing to change global defaults may lead to surprises when there are multiple people working on the project or when you forget about the change after some time. Such functionality could be useful for people migrating their projects from go-pg, but I've not heard many complaints so far. Personally, I've ended up not using `nullzero` as much as I've used it before. There are pros and cons, but it does not seem to matter much.
